presto-product-tests/bin/locations.sh

```diff
@@ -2,3 +2,16 @@ SCRIPT_DIR=${BASH_SOURCE%/*}
 PRODUCT_TESTS_ROOT="${SCRIPT_DIR}/.."
 PROJECT_ROOT="${PRODUCT_TESTS_ROOT}/.."
 DOCKER_CONF_LOCATION="${PRODUCT_TESTS_ROOT}/conf/docker"
+
+# Yes, JDBC_DRIVER_JAR is intentionally set to a directory. 
+# This works better than leaving it empty or unset.
```

Comment:
It's not necessarily a jar, it's a classpath entry (like the PRODUCT_TEST_ROOT you use here).
Also, it does not say which jdbc driver it is. Please rename the variable PRESTO_JDBC_DRIVER_CLASSPATH

EDIT: actually, it should always be a jar, because presto-tests does not have the product tests's `target` dir, just the jar. So let's name it PRESTO_JDBC_DRIVER_JAR and default to $PRODUCT_TEST_JAR

EDIT2: that's another reason to put this variable in compose-commons, as that's the place PRODUCT_TEST_JAR has its default defined.